packages/shared/src/components/squads/SquadHeaderBar.tsx

```diff
@@ -81,15 +72,153 @@ export function SquadHeaderBar({
     return null;
   }, [sourceIntegration, user, squad, isPending]);
 
+  if (!slackButtonLabel) {
+    return null;
+  }
+
+  return (
+    <Button
+      variant={ButtonVariant.Secondary}
+      icon={<SlackIcon />}
+      size={ButtonSize.Small}
+      {...props}
+    >
+      {slackButtonLabel}
+    </Button>
+  );
+};
+
+const SquadInviteButton = <T extends AllowedTags>({
+  squad,
+  ...props
+}: SquadBarButtonProps<T>) => {
+  const canRender = useMemo(() => {
+    return verifyPermission(squad, SourcePermissions.Invite);
+  }, [squad]);
+
+  if (!canRender) {
+    return null;
+  }
+
+  return (
+    <Button
+      variant={ButtonVariant.Secondary}
+      size={ButtonSize.Small}
+      icon={<AddUserIcon />}
+      {...props}
+    >
+      Invitation link
+    </Button>
+  );
+};
+
+const SquadChecklistButton = ({ squad }: SquadBarButtonProps<'button'>) => {
+  const {
+    steps,
+    completedSteps,
+    isChecklistVisible,
+    setChecklistVisible,
+    isChecklistReady,
+  } = useSquadChecklist({ squad });
+
+  const completedStepsCount = completedSteps.length;
+  const totalStepsCount = steps.length;
+  const checklistTooltipText = `${completedStepsCount}/${totalStepsCount}`;
+
+  return (
+    <SimpleTooltip
+      forceLoad={!isTesting}
+      visible={isChecklistReady && completedStepsCount < totalStepsCount}
+      container={{
+        className: '-mb-4 !bg-accent-onion-default !text-white',
+      }}
+      placement="top"
+      content={checklistTooltipText}
+      zIndex={3}
+    >
+      <Button
+        data-testid="squad-checklist-button"
+        variant={ButtonVariant.Float}
+        icon={<ChecklistBIcon secondary />}
+        onClick={() => {
+          setChecklistVisible(!isChecklistVisible);
+        }}
+        size={ButtonSize.Small}
+      />
+    </SimpleTooltip>
+  );
+};
+
+const SquadUserNotifications = ({
+  squad,
+  ...props
+}: SquadBarButtonProps<'button'>) => {
+  return (
+    <SimpleTooltip
+      forceLoad={!isTesting}
+      placement="bottom"
+      content="Squad notifications settings"
+    >
+      <Button
+        data-testid="squad-notification-button"
+        className="order-3 tablet:order-4"
+        variant={ButtonVariant.Float}
+        size={ButtonSize.Small}
+        {...props}
+      />
+    </SimpleTooltip>
+  );
+};
+
+const SquadModerationButton = ({ squad }: SquadBarButtonProps<'a'>) => {
+  if (!squad.moderationRequired || !squad.moderationPostCount) {
+    return null;
+  }
+
+  const count = squad.moderationPostCount;
+  const postLabel = count === 1 ? 'post' : 'posts';
+
+  return (
+    <Button
+      aria-label={`Go to post moderation page and check ${count} ${postLabel}`}
```

Comment:
Would just change to `Review ${count} ${postLabel}` since navigation is implied with it being an a-tag.

Reply:
Good point, changing to `Check ${count} pending ${postLabel}` because is both for mod/users